package-lock.json

```diff
@@ -0,0 +1,9898 @@
+{
```

Comment:
Did you run any command with `npm`?. This project uses `yarn`, not `npm`. I think this file shouldn't have been generated.